x-pack/plugins/data_visualizer/public/application/common/components/date_picker_wrapper/date_picker_wrapper.tsx

```diff
@@ -242,9 +244,17 @@ export const DatePickerWrapper: FC<{ isAutoRefreshOnly?: boolean; showRefresh?:
     <EuiFlexGroup
       gutterSize="s"
       alignItems="center"
-      className="mlNavigationMenu__datePickerWrapper"
+      data-test-subj="mlNavigationMenuDatePickerWrapper"
     >
-      <EuiFlexItem grow={false}>
+      <EuiFlexItem
+        grow={false}
+        css={
+          compact &&
+          css`
+            max-width: 540px;
+          `
+        }
```

Comment:
For the EUI components, `css` helper is redundant. You can just pass an object to the `css` prop.
```suggestion
        css={compact ? {
            maxWidth: '540px',
          } : null}
```

Reply:
Updated here [`2d35742` (#147137)](https://github.com/elastic/kibana/pull/147137/commits/2d35742e343066904b17034fa60a0b1c3fb5536e)